### ISSUE TITLE
Go-to definition with cmd+click or f12 not working

### ISSUE THREAD

**Issue by Prakyathkantharaju** [cursor/cursor#1493]
1) If you can, please include a screenshot of your problem
This issue was previously reported: the go-to definition, which is a right-click option in vscode, is missing in the cursor. This also includes the `F12` keyboard shortcut. This issue is previously reported in the forum for more than 3 months. This also does not work in the vim mode, `gd`

2) Please include the name of your operating system
I have tried in linux and the max-os the problem persists.

3) If you can, steps to reproduce are super helpful
`Nan`

**Comment by xiazhi1** [comment]
same problem, I think this significantly reduces the usability of the cursor

**Comment by sakshamjindal** [comment]
Install "Python" and "Pylance" extensions. Now it is working. This will set your python language server to pylance.

**Comment by 37huang** [comment]
how to fix ?

**Comment by acdameli** [comment]
Yeah, I am encountering this issue as well it's impacting my custom keyboard shortcut of ctrl+B I'm getting "peek definition" instead of "goto definition".

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
